src/main/java/org/isf/medicalinventory/gui/InventoryWardEdit.java

```diff
@@ -184,31 +218,35 @@ private void fireInventoryUpdated() {
             .getBean(MovWardBrowserManager.class);
     private MovStockInsertingManager movStockInsertingManager = Context.getApplicationContext()
             .getBean(MovStockInsertingManager.class);
+    private MedicalDsrStockMovementTypeBrowserManager medStockMovTypeManager = Context.getApplicationContext()
+            .getBean(MedicalDsrStockMovementTypeBrowserManager.class);
+    private SupplierBrowserManager supplierBrowserManager = Context.getApplicationContext().getBean(SupplierBrowserManager.class);
+
 
     public InventoryWardEdit() {
         mode = "new";
         initComponents();
-        cancelButton.setVisible(false);
+        resetButton.setVisible(false);
         disabledSomeComponents();
     }
 
     public InventoryWardEdit(MedicalInventory inventory, String modee) {
         this.inventory = inventory;
         mode = modee;
         initComponents();
-        if (mode.equals("view")) {
-            validateButton.setVisible(false);
-            saveButton.setVisible(false);
-            cancelButton.setVisible(false);
-            wardComboBox.setEnabled(false);
-        }
     }
 
     private void initComponents() {
         setDefaultCloseOperation(JFrame.DISPOSE_ON_CLOSE);
         setMinimumSize(new DimensionUIResource(950, 580));
         setLocationRelativeTo(null); // center
-        setTitle(MessageBundle.getMessage("angal.inventory.edit.title"));
+        if (mode.equals("update")) {
+            setTitle(MessageBundle.getMessage("angal.inventory.edit.title"));
+        } else if (mode.equals("view")) {
+            setTitle(MessageBundle.getMessage("angal.inventory.view.title"));
+        } else {
+            setTitle(MessageBundle.getMessage("angal.inventory.new.title"));
```

Comment:
This key isn't in the resource bundle file; but there is `angal.inventory.newinventory.title` at line 705